modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/credential/CredentialCenter.java

```diff
@@ -278,23 +279,10 @@ private CredentialVariables createCredentialVariablesObject(
                 createCredential.getXpanseUser(),
                 createCredential.getName(), createCredential.getDescription(),
                 createCredential.getType(), createCredential.getVariables());
-        if (Objects.isNull(createCredential.getTimeToLive())) {
-            createCredential.setTimeToLive(DEFAULT_TIMEOUT_SECONDS);
-        }
-        credential.setExpiredTime(
-                System.currentTimeMillis() + createCredential.getTimeToLive() * 1000);
+        credential.setTimeToLive(Integer.toUnsignedLong(createCredential.getTimeToLive()));
```

Comment:
We can unify the type of  the filed 'timeToLive' in both classes AbstractCredentialInfo and CreateCredential.

Reply:
fixed